aeneas/src/vst/Verifier.v3

```diff
@@ -886,29 +893,61 @@ class VstCompoundVerifier {
 						_ => ;
 					}
 				}
-				if (rep.length > bitWidth) errAtToken(token).BitPatternTooLong(bitWidth, rep.length);
-			}
-			Tuple(list) => {
-				for (l = list.list; l != null; l = l.tail) {
-					checkPackingExpr(env, l.head, MAX_PACKING_WIDTH); // TODO: proper size
-				}
-			}
-			App(p, args) => { // TODO
-				// TODO: resolve and apply packing {p}
-				for (l = args.list; l != null; l = l.tail) {
-					checkPackingExpr(env, l.head, MAX_PACKING_WIDTH); // TODO: proper size
+				return byte.!(rep.length);
```

Comment:
A naked cast here is somewhat dangerous (could crash the compiler, even if an error is reported elsewhere). Better to use a `view` or clamp.